teacher.rb

```diff
@@ -26,15 +18,6 @@ def teach_stuff
     response
   end
 
-  def salary=(new_salary)
-    puts "This better be good!"
-    @salary = new_salary
-  end
-
-  def receive_raise(raise)
-    @salary += raise
-  end
-
   def set_performance_rating(rating)
```

Comment:
This looks really similar to a method in the `ApprenticeTeqacher` class with just one difference.  How could you DRY up this repeated code?